tests/integration/models/fields_test.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+from unittest import TestCase
+from datetime import datetime as dt
+
+from django.db import models
+
+from nav.models.msgmaint import MaintenanceComponent, MaintenanceTask
+from nav.models.event import Subsystem
+from nav.models.logger import ErrorError
+from nav.models.fields import LegacyGenericForeignKey
+
+
+class LegacyGenericForeignKeyTest(TestCase):
```

Comment:
You should really be inheriting `nav.tests.cases.DjangoTransactionTestCase` here, or you'll litter the test database.

Reply:
willdo! 

I've discovered why the check mark is on "make newer" and not "add tests" though. Github sorts by date, not order of commits, and I moved "make newer" (older than "add tests") to the end/HEAD in order to make it easier to revert if necessary.